### ISSUE TITLE
bug: WalletConnect connector reporting invalid chain ID after activation

### ISSUE THREAD

**Issue by jariz** [NoahZinsmeister/web3-react#370]
## Reproduction steps

- Switch your wallet to a different network than the default one (I used metamask mobile)
- Go to the example: https://web3-react-mu.vercel.app/
- Sign in with WalletConnect
- The page now says 'active ID: 1', which is incorrect
- Refresh the page
- The page now says active ID: `<whatever you selected in your wallet>`

So, refreshing the page seems to be a workaround, however, other providers do not have this problem and report the correct ID directly after activation.

## Versions used
```
    "@web3-react/core": "^8.0.1-beta.0",
    "@web3-react/types": "^8.0.0-beta.0",
    "@web3-react/walletconnect": "^8.0.1-beta.0",
```

**Comment by NoahZinsmeister** [comment]
almost certainly a walletconnect bug, but will take a look! thanks for reporting

**Comment by NoahZinsmeister** [comment]
fixed as of 8.0.2-beta.0!

**Comment by jariz** [comment]
Thanks for the quick fix! ❤️
